src/vivarium_nih_us_cvd/components/causes.py

```diff
@@ -0,0 +1,163 @@
+from typing import Any, Callable, Dict, List, Union
+
+import pandas as pd
+from vivarium import ConfigTree
+from vivarium.framework.engine import Builder
+from vivarium_public_health.disease import (
+    BaseDiseaseState,
+    DiseaseModel,
+    DiseaseState,
+    RecoveredState,
+    SusceptibleState,
+)
+from vivarium_public_health.utilities import TargetString
+
+from vivarium_nih_us_cvd.constants.paths import CAUSE_RISK_CONFIG
+
+
+class Causes:
+    # todo could modify the component configuration parser plugin
+    def __init__(self):
+        self._sub_components = self.get_disease_models()
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def name(self) -> str:
+        return "causes"
+
+    @property
+    def sub_components(self) -> List:
+        return self._sub_components
+
+    ##################
+    # Helper methods #
+    ##################
+
+    def get_disease_models(self) -> List:
+        disease_models = []
+
+        full_config = ConfigTree(layers=["default", "model_spec"])
+        full_config.update(CAUSE_RISK_CONFIG, layer="model_spec")

```

Comment:
We should maybe re-think these layer names now since both yamls make up the "model spec"

Reply:
You could reasonably argue that the causes.yaml is not the[ model_spec, as currently defined.](https://vivarium.readthedocs.io/en/latest/concepts/model_specification/index.html)